src/main/java/org/zalando/nakadi/controller/EventTypeController.java

```diff
@@ -60,6 +66,10 @@ public EventTypeController(final EventTypeService eventTypeService,
         }
 
         ValidationUtils.invokeValidator(eventTypeOptionsValidator, eventType.getOptions(), errors);
+        if (!applicationService.exists(eventType.getOwningApplication())) {
```

Comment:
I think we need a future-toggle for this feature as it forbids something that was allowed in the past.